tests/gtest/avif_fuzztest_properties.cc

```diff
@@ -74,22 +85,14 @@ inline auto ArbitraryProp() {
   auto fourcc = fuzztest::Arbitrary<std::array<uint8_t, 4>>();
   auto uuid = fuzztest::Arbitrary<std::array<uint8_t, 16>>();  // ignored
   auto body = fuzztest::Arbitrary<std::vector<uint8_t>>();
-  // Don't return known properties.
-  return fuzztest::Filter(
-      [](const TestProp& prop) {
-        return !avifIsKnownPropertyType(prop.fourcc.data());
```

Comment:
A random fourcc is unlikely to be a known property type. So this Filter should be okay.